README.md

```diff
@@ -52,7 +52,7 @@ or simply from GitHub:
 
 ```bash
 git clone https://github.com/quandela/Perceval
-python setup.py install # [or 'develop' for developers]
```

Comment:
I just have one last thing: the comment `# [or 'develop' for developers]` should turn into : 
`# or pip install -e . for developers`

Could you please add this mention back at both places you updated the install command?
Then the PR is good!

Reply:
done !